gui:
  ros__parameters:
    show_axes: false
    show_curvatures: false
    show_fov_clusters: false
    show_grid: false
    show_mesh: true
    show_model_bounding_box: false
    show_noise_points: false
    show_path: false
    show_point_cloud: false
    show_region_view_manifolds: false
    show_regions: false
    show_reticle: true
    show_skybox: true
    show_viewpoints: false
    use_sim_time: false

inspection_task_planning:
  ros__parameters:
    viewpoints_file: "$INSPECTION_WS/src/ViewpointGenerationData/cylinder_point_cloud/cylinder_mm_point_cloud_100000points_0.1_100_100000_0.1_0.1_0.5235987755982988_2025-08-19_15-43-40_viewpoints_optimized2025-08-19_15-49-25.json"

viewpoint_generation:
  ros__parameters:
    model.camera.dof: 0.02
    model.camera.focal_distance: 0.3
    model.camera.fov.height: 0.1
    model.camera.fov.width: 0.1
    model.mesh.file: "$INSPECTION_WS/src/ViewpointGenerationData/cylinder.stl"
    model.mesh.units: mm
    model.point_cloud.file: "$INSPECTION_WS/src/ViewpointGenerationData/cylinder_point_cloud/cylinder_mm_point_cloud_100000points.ply"
    model.point_cloud.sampling.number_of_points: 100000
    model.point_cloud.sampling.ppsqmm: 2.341362487277471
    model.point_cloud.units: m
    regions.file: "$INSPECTION_WS/src/ViewpointGenerationData/cylinder_point_cloud/cylinder_mm_point_cloud_100000points_0.1_100_100000_0.1_0.1_0.5235987755982988_2025-08-19_15-43-40_viewpoints_optimized2025-08-19_15-49-25.json"
    regions.fov_clustering.beta_weight: 1.0
    regions.fov_clustering.k-means.maximum_iterations: 100
    regions.fov_clustering.k-means.normal_weight: 1.0
    regions.fov_clustering.k-means.number_of_runs: 10
    regions.fov_clustering.k-means.point_weight: 1.0
    regions.fov_clustering.lambda_weight: 1.0
    regions.fov_clustering.max_point_out_percentage: 0.001
    regions.region_growth.curvature.file: ''
    regions.region_growth.curvature.knn_neighbors: 10
    regions.region_growth.curvature_threshold: 0.1
    regions.region_growth.max_cluster_size: 100000
    regions.region_growth.min_cluster_size: 100
    regions.region_growth.normal_angle_threshold: 60.0
    regions.region_growth.seed_threshold: 0.1
    settings.cuda_enabled: false
    use_sim_time: false
    viewpoints.projection.nothing: ''
    viewpoints.traversal: ''
